lib/index.js

```diff
@@ -55,6 +57,25 @@ exports.plugin = {
             server.ext('onPreStart', internals.initialize);
             server.ext('onPostStop', internals.stop);
 
+            // Expose an hpal command for schwifty-migration
+            server.expose('commands', {
+                migrations: (initializedServer, args) => {
+
+                    // args order is [mode, migrationName, migrationsDir]
+
+                    const models = initializedServer.models();
+                    const modelsArr = Object.keys(models).map((key) => models[key]);
+
+                    return promisify(SchwiftyMigration.genMigrationFile)({
+                        models: modelsArr,
```

Comment:
Some of these models may not be using `initializedServer.knex()`—we should think about how to best handle that.

Reply:
Keep in mind we can have schwifty provide whatever info is needed to run these migrations.  It's awkward that it's hard to target a specific plugin's knex instance.